web/package.json

```diff
@@ -12,6 +12,7 @@
     "bs-ocaml-protoc-json": "^0.1.3",
     "bs-parse": "git+https://github.com/aitoroses/bs-parse.git",
     "chart.js": "^2.9.4",
+    "chokidar": "^3.5.3",
```

Comment:
That's not correct because this is not a direct dependencies of monocle. To get a new version of that library, you need to find what is requiring it, and update the minimum version of one of the existing dependency.

Reply:
I see that now. I will update the PR with changing the base package